Integrations/ML/UI/Panels/modallenspanel.lua

```diff
@@ -9,7 +9,18 @@ include( "CQUICommon.lua" );
 -- ===========================================================================
 
 g_ModLensModalPanel = {} -- Populated by ModLens_*.lua scripts
-include( "ModLens_", true )
+--include( "ModLens_", true )
+-- Need to do these individually, instead of using the include("ModLens_", true), because it appears on Linux ModLens_Builder_Config_Default.lua would get loaded before ModLens_Builder.lua (when ModLens_Builder.lua includes _Builder_Config_Default).
+-- Casing also matters for these filenames with Linux: these ModLens_ files are checked in to the repo with the casing seen below.
+include("ModLens_Archaeologist.lua");
```

Comment:
It cannot be like this. Our goal is to make ML work on Linux with the default features. And the key feature is the ability to load multiple lenses, also for compatibility with other mods. So, such iterated list is a no-go.

Reply:
Ok, not going to argue with that. I was really just trying to see what would/wouldn't work from the original branch rebased to master.
Having the minimap without all the extra lenses is a step forward from having no minimap and no lenses though!  :-)

Reply:
Well, you raised a PR, hence the comment...